inc/sitemaps/class-sitemaps.php

```diff
@@ -356,7 +352,7 @@ public function build_root_map() {
 		$links = array();
 
 		foreach ( $this->providers as $provider ) {
-			$links = array_merge( $links, $provider->get_index_links( $this->max_entries ) );
+			$links = array_merge( $links, $provider->get_index_links( $this->get_entries_per_page() ) );
```

Comment:
Maybe it's a tiny optimization, but I would save the output of `get_entries_per_page` outside of the `foreach`